test/controllers/session_controller_test.exs

```diff
@@ -0,0 +1,38 @@
+defmodule ResearchResource.SessionControllerTest do
+  use ResearchResource.ConnCase
+
+  setup %{conn: conn} = config do
+    if username = config[:login_as] do
```

Comment:
If `login_as` doesn't exist in `config`, calling `config.login_as` will throw a `KeyError`, while `config[:login_as]` will just return nil. Because we're using it in an if statement here, we want to just execute the `else` branch if `login_as` doesn't exist